pages/404.js

```diff
@@ -0,0 +1,31 @@
+import Image from 'next/image';
+import React from 'react';
+
+import ImageGif404 from '../public/images/404.gif';
+export default function NotFound() {
+  return (
+    <html>
+      <head>
+        <title> {'404 Not Found'} </title>
```

Comment:
remove the extra spaces before `{` and after `}` - what IDE are you using? If Visual Studio code than you can install `Prettier` extension and set to format code on saving file.